docs/tutorial/create_contact_page.md

```diff
@@ -65,6 +65,8 @@ class FormPage(AbstractEmailForm):
     ]
 ```
 
+**Note** : For local development, it is recommended to use django.core.mail.backends.console.EmailBackend, which will log emails to the console rather than sending them. For production, configure an SMTP email backend to send form submissions via a third-party email service. Fly.io does not provide email services out of the box, so you’ll need to set up a third-party SMTP provider like SendGrid.
```

Comment:
I think this would be better below, where we talk about "form-to-email capability". As-is, it’s not clear why this note is relevant.

Reply:
Thank you for your suggestion! I'll move it to its respective section.